src/ixdat/exporters/ecms_exporter.py

```diff
@@ -1,36 +1,67 @@
-from .csv_exporter import CSVExporter
 from .ec_exporter import ECExporter
+from ..tools import deprecate
 
 
-class ECMSExporter(CSVExporter):
+class ECMSExporter(ECExporter):
     """A CSVExporter that by default exports potential, current, selector, and all MID"""
 
     @property
     def default_export_columns(self):
         """The default v_list for ECExporter is V_str, J_str, and sel_str"""
         v_list = (
-            ECExporter(measurement=self.measurement).default_v_list
+            ECExporter(measurement=self.measurement).default_export_columns
             + self.measurement.mass_list
         )
 
         return v_list
 
+    @deprecate("0.2.0", "use `columns` instead", "0.3.0", kwarg_name="v_list")
     def export(
         self,
         path_to_file=None,
         measurement=None,
         v_list=None,
+        columns=None,
         tspan=None,
         mass_list=None,
         mol_list=None,
+        time_step=None,
     ):
-        if not v_list:
+        """Export a given measurement to a specified file.
+
+        To improve flexibility with inheritance, this method allocates its work to:
+        - CSVExporter.prepare_header_and_data()
+        - CSVExporter.write_header()
+        - CSVExporter.write_data()
+
+        Args:
+            measurement (Measurement): The measurement to export.
+                Defaults to self.measurement.
+                TODO: remove this kwarg. See conversation here:
+                   https://github.com/ixdat/ixdat/pull/30/files#r810926968
+            path_to_file (Path): The path to the file to write. If it has no suffix,
+                a .csv suffix is appended. Defaults to f"{measurement.name}.csv"
+            columns (list of str): The names of the data series to include. Defaults in
```

Comment:
I think this one is a little generic. If you want to mention what it defaults to, you should probably (if it is possible) mention what it defaults to _in the case of an ECMS exporter_